WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -257,11 +257,6 @@ public void run() {
             }
 
             if (FluxCUtils.isSignedInWPComOrHasWPOrgSite(mAccountStore, mSiteStore)) {
-                NotificationType notificationType =
-                        (NotificationType) getIntent().getSerializableExtra(ARG_NOTIFICATION_TYPE);
-                if (notificationType != null) {
-                    mSystemNotificationsTracker.trackTappedNotification(notificationType);
-                }
                 // open note detail if activity called from a push
```

Comment:
Maybe I'm missing something, but I think we should not remove this one. If I'm not making things wrong, I tried triggering a notification for a comment related to a followed site and in that case I do not get the tap track I suspect because we removed this one; getting something like:

```
🔵 Tracked: notification_tapped_segmented_control, Properties: {"selected_filter":"all"}
🔵 Tracked: notification_tapped_segmented_control, Properties: {"selected_filter":"all"}
```

Expecting something like:

```
🔵 Tracked: notification_tapped, Properties: {"notification_type":"comment"}
🔵 Tracked: notification_tapped_segmented_control, Properties: {"selected_filter":"all"}
🔵 Tracked: notification_tapped_segmented_control, Properties: {"selected_filter":"all"}
```

Reply:
totally spot on @develric - more on this in https://github.com/wordpress-mobile/WordPress-Android/pull/12184#discussion_r453871238